Crear proyecto nuevo con Nextjs

MathiMartinez00 opened the pull request:
Se realizó la creación del proyecto según lo especificado en la tarea [JOKO-46](https://sodepsa.atlassian.net/browse/JOKO-46). Las opciones elegidas durante la creación de este están listadas en README.md

MathiMartinez00 committed:
Crear proyecto Next.js

El proyecto fue creado con las siguientes opciones:
- TypeScript: Sí.
- ESLint: Sí.
- Tailwind CSS: No.
- Utilizar el directorio src/: Sí.
- Utilizar el direction app/: No.
- Import alias: @/*.

MathiMartinez00 committed:
Instalar libreria antd

rodrigojv reviewed (commented):
Buenísimo @MathiMartinez00 

Pensándolo bien, es mejor utilizar Tailwind en lugar de AntD.

Entonces para completar este PR:
1. Agregar tailwind. Seguir estos pasos: https://tailwindcss.com/docs/guides/nextjs. Realizar los cambios mencionados ahi sobre `src/pages/index.tsx`
2. Eliminar contenido de `styles/globals.css` y dejar solamente lo que menciona la guia del paso 1.
3. Una vez completado el paso 1 y 2, eliminar styles/Home.module.css
4. Eliminar dependencia de AntD del package.json

Commit:
Instalar Tailwind CSS y desintalar antd

Se realizaron las modificaciones sugeridas por Rodrigo que incluyen:
Instalar Tailwind CSS lo cual agrega los siguientes archivos de configuracion:
- postcss.config.js.
- tailwind.config.js.
Por ende, se vieron modificados los siguientes archivos:
- package.json: Agregar dependencias y eliminar antd.
- package-lock.json: Agregar dependencias y eliminar antd.
- src/styles/globals.css: Cambiar los contenidos para el funcionamiento con Tailwind.
- src/pages/index.tsx: Replicar el ejemplo de Hello World segun la guia de Tailwind.
Por ultimo, se elimina el siguiente archivo:
- src/styles/Home.module.css.
